squad/ci/backend/lava.py

```diff
@@ -205,10 +206,27 @@ def __download_full_log__(self, job_id):
         response = requests.get(url, params=payload)
         return response.content
 
-    def __get_job_logs__(self, job_id):
-        logger.debug("Retrieving logs job: %s" % job_id)
-        log_data = self.__download_full_log__(job_id)
-        return self.__parse_log__(log_data)
+    def __download_test_log__(self, raw_log, log_start, log_end):
+        return_lines = ""
+        if not log_start:
+            return return_lines
+        log_start_line = int(log_start)
+        log_end_line = None
+        if log_end:
+            log_end_line = int(log_end)
+        else:
+            log_end_line = log_start_line + 2  # LAVA sometimes misses the signals
+        raw_log.seek(0)
+        counter = 0
+        for line in raw_log:
+            counter += 1
+            if counter <= log_start_line:
```

Comment:
I think this line should be `counter < log_start_line`, otherwise the first log line will always be missing.

Reply:
I wasn't sure about that but you might be right. 1st line is usually not that important anyway but I'll change it.